projects/packages/changelogger/tests/changelogger

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env php
```

Comment:
I think this should be a test, not a copy, right? :^)

Reply:
No, it's a copy with the `require $GLOBALS['_composer_autoload_path'];` changed to `require __DIR__ . '/../vendor/autoload.php';`. Unfortunately composer doesn't install the local package's bin files into `vendor/bin/`, so there's no way to easily get that set for running it locally instead of having `composer install` install it somewhere else.

If this location is confusing, maybe I should just move it to something like `bin/local-changelogger` and add a `.gitattributes` entry to exclude it from the mirror.

Reply:
Or I guess I could have the normal bin do `require_once $GLOBALS['_composer_autoload_path'] ?? __DIR__ . '/../vendor/autoload.php';`, but then that'd be there for everyone even though it's only correct for running from the monorepo source dir.